source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -394,7 +394,11 @@ def _get_name(self):
 				textList.append("%s: %s" % (header, content))
 			else:
 				textList.append(content)
-		return "; ".join(textList)
+		name = "; ".join(textList)
+		#Some list view items in Vista and later can contain annoying left-to-right and right-to-left indicator characters which really should not be there.
+		name=name.replace(u'\u200E','')
+		name=name.replace(u'\u200F','')

```

Comment:
Actually, it'd be more efficient to use unicode.translate, since you can 
do it all in one call:

name = name.translate(({0x200e: None, 0x200f: None})

If we use a similar pattern elsewhere in that file, feel free to change 
that too. Note that this is only more efficient if you're calling 
replace more than once and only with single characters.